test/fixtures/requests/getOrderbookWithXrp.json

```
@@ -1,9 +1,9 @@
 {
-  "base": {
+  "taker_pays": {
```

```suggestion
  "takerPays": {
```
We should be consistent everywhere